frontend/src/pages/acceleratorProfile/AcceleratorProfile.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import {
+  Button,
+  ButtonVariant,
+  Flex,
+  FlexItem,
+  EmptyState,
+  EmptyStateIcon,
+  EmptyStateVariant,
+  EmptyStateBody,
+  PageSection,
+  PageSectionVariants,
+  Title,
+} from '@patternfly/react-core';
+import { PlusCircleIcon } from '@patternfly/react-icons';
+import ApplicationsPage from '~/pages/ApplicationsPage';
+import useAccelerators from '~/pages/notebookController/screens/server/useAccelerators';
+import { useDashboardNamespace } from '~/redux/selectors';
+
+const description = `Manage accelerator profile settings for users in your organization`;
+
+const AcceleratorProfile: React.FC = () => {
+  const { dashboardNamespace } = useDashboardNamespace();
+  const [accelerators, loaded, loadError] = useAccelerators(dashboardNamespace);
+
+  const isEmpty = !accelerators || accelerators.length === 0;
+
+  const noAcceleratorProfilePageSection = (
+    <PageSection isFilled>
+      <EmptyState variant={EmptyStateVariant.full} data-id="empty-empty-state">
+        <EmptyStateIcon icon={PlusCircleIcon} />
+        <Title headingLevel="h5" size="lg">
+          No available accelerator profiles yet
+        </Title>
+        <EmptyStateBody>
+          {/* eslint-disable-next-line react/no-unescaped-entities */}
+          You don't have any accelerator profiles yet. To get started, please ask your cluster
+          administrator about
+          <br /> the accelerator availability in your cluster and create corresponding profiles in
+          Openshift Data Science.
+        </EmptyStateBody>
+        <Button
+          data-id="display-accelerator-modal-button"
+          variant={ButtonVariant.primary}
+          // eslint-disable-next-line @typescript-eslint/no-empty-function
+          onClick={() => {}}
```

Comment:
Same as above, we should not have to overwrite eslint. Setting as undefined works fine.

```suggestion
          onClick={() => undefined}
```